src/Brouter/Demo/Bit.Brouter.Demo.Core/wwwroot/app.css

```
@@ -0,0 +1,421 @@
+    --bb-primary: #4f46e5;
+    --bb-primary-hover: #4338ca;
+    --bb-primary-soft: #eef2ff;
+    --bb-accent: #0ea5e9;
+    --bb-success: #10b981;
+    --bb-warning: #f59e0b;
+    --bb-danger: #ef4444;
+    --bb-radius: 10px;
+    --bb-radius-sm: 6px;
+    --bb-shadow-sm: 0 1px 2px rgba(15, 23, 42, .06);
+    --bb-shadow: 0 6px 20px rgba(15, 23, 42, .06);
+    --bb-mono: ui-monospace, SFMono-Regular, "SF Mono", Menlo, Consolas, "Liberation Mono", monospace;
```

Stylelint (17.12.0)

[error] Expected "SFMono-Regular" to be "sfmono-regular" (value-keyword-case)

(value-keyword-case)